pkg/scalers/connectionpool/config.go

```diff
@@ -0,0 +1,97 @@
+package connectionpool
```

Comment:
I prefer to not read extra configMaps tbh. Are so many parameters needed? can we pass the via cmd args during the startup?